include/revng/Model/model-schema.yml

```diff
@@ -2,7 +2,7 @@
 # This file is distributed under the MIT License. See LICENSE.md for details.
 #
 
-version: 1
```

Comment:
There's version `2` which does nothing, drop version `3`

Reply:
I actually jumped to 3 on purpose, I think it's good if we test with more than a single migration step.

Would you prefer to make a dummy change to this version as well?

Reply:
> I think it's good if we test with more than a single migration step

Ah ok, that's good, keep it